Makefile

```diff
@@ -256,12 +256,12 @@ ifeq ($(DOCKER),docker)
 	# docker uses builder containers
 	- $(DOCKER) buildx rm model-registry-builder
 	$(DOCKER) buildx create --use --name model-registry-builder --platform=$(PLATFORMS)
-	$(DOCKER) buildx build --push --platform=$(PLATFORMS) --tag ${IMG} -f ${DOCKERFILE} .
+	$(DOCKER) buildx build --push --platform=$(PLATFORMS) --tag ${IMG}:$(IMG_VERSION) -f ${DOCKERFILE} .
```

Comment:
Makes it consistent with 

https://github.com/kubeflow/model-registry/blob/4099d46f35a8d5a0f5e8c5f357c39545f60f5433/Makefile#L241-L243

lgtm